hurumap_ng/sql/suspects_prosecuted.sql

```diff
@@ -0,0 +1,243 @@
+state	13	2016	2014	19
+state	13	2016	2015	25
+state	13	2016	2016	24
+state	14	2016	2012	70
+state	14	2016	2013	26
+state	14	2016	2014	18
+state	14	2016	2015	13
+state	14	2016	2016	61
+state	15	2016	2012	122
+state	15	2016	2013	77
+state	15	2016	2014	115
+state	15	2016	2015	95
+state	15	2016	2016	146
+state	16	2016	2012	58
+state	16	2016	2013	37
+state	16	2016	2014	48
+state	16	2016	2015	58
+state	16	2016	2016	95
+state	17	2016	2012	80
+state	17	2016	2013	22
+state	17	2016	2014	24
+state	17	2016	2015	18
+state	17	2016	2016	5
+state	18	2016	2012	80
+state	18	2016	2013	61
+state	18	2016	2014	62
+state	18	2016	2015	10
+state	18	2016	2016	84
+state	19	2016	2012	57
+state	19	2016	2013	65
+state	19	2016	2014	151
+state	19	2016	2015	29
+state	19	2016	2016	33
+state	20	2016	2012	192
+state	20	2016	2013	217
+state	20	2016	2014	182
+state	20	2016	2015	156
+state	20	2016	2016	343
+state	21	2016	2012	113
+state	21	2016	2013	79
+state	21	2016	2014	104
+state	21	2016	2015	72
+state	21	2016	2016	16
+state	22	2016	2012	118
+state	22	2016	2013	108
+state	22	2016	2014	59
+state	22	2016	2015	42
+state	22	2016	2016	56
+state	23	2016	2012	37
+state	23	2016	2013	34
+state	23	2016	2014	16
+state	23	2016	2015	22
+state	23	2016	2016	20
+state	24	2016	2012	29
+state	24	2016	2013	24
+state	24	2016	2014	17
+state	24	2016	2015	13
+state	24	2016	2016	17
+state	25	2016	2012	123
+state	25	2016	2013	76
+state	25	2016	2014	125
+state	25	2016	2015	143
+state	25	2016	2016	211
+state	26	2016	2012	5
+state	26	2016	2013	2
+state	26	2016	2014	22
+state	26	2016	2015	29
+state	26	2016	2016	36
+state	27	2016	2012	43
+state	27	2016	2013	46
+state	27	2016	2014	24
+state	27	2016	2015	46
+state	27	2016	2016	22
+state	28	2016	2012	36
+state	28	2016	2013	51
+state	28	2016	2014	31
+state	28	2016	2015	19
+state	28	2016	2016	18
+state	29	2016	2012	30
+state	29	2016	2013	29
+state	29	2016	2014	31
+state	29	2016	2015	36
+state	29	2016	2016	44
+state	30	2016	2012	119
+state	30	2016	2013	80
+state	30	2016	2014	60
+state	30	2016	2015	28
+state	30	2016	2016	47
+state	31	2016	2012	65
+state	31	2016	2013	65
+state	31	2016	2014	55
+state	31	2016	2015	56
+state	31	2016	2016	64
+state	32	2016	2012	115
+state	32	2016	2013	122
+state	32	2016	2014	91
+state	32	2016	2015	82
+state	32	2016	2016	96
+state	33	2016	2012	13
+state	33	2016	2013	60
+state	33	2016	2014	86
+state	33	2016	2015	51
+state	33	2016	2016	64
+state	34	2016	2012	22
+state	34	2016	2013	28
+state	34	2016	2014	18
+state	34	2016	2015	11
+state	34	2016	2016	33
+state	35	2016	2012	26
+state	35	2016	2013	16
+state	35	2016	2014	39
+state	35	2016	2015	32
+state	35	2016	2016	53
+state	36	2016	2012	12
+state	36	2016	2013	7
+state	36	2016	2014	10
+state	36	2016	2015	0
+state	36	2016	2016	19
+state	37	2016	2012	23
+state	37	2016	2013	42
+state	37	2016	2014	36
+state	37	2016	2015	41
+state	37	2016	2016	56
+country	NG	2016	2012	2292
+country	NG	2016	2012	1772
+country	NG	2016	2012	1981
+country	NG	2016	2012	1633
+country	NG	2016	2012	2194
```

Comment:
Year `2012` repeated 5 times causing `could not create unique index "suspects_prosecuted_pkey"` error. You probably meant `2012, 2013, 2014, 2015, and 2016`